lib/src/view/windows.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/widgets.dart';
+import 'package:webview_windows/webview_windows.dart';
+import 'package:webviewx/src/controller/windows.dart';
+
+import 'device.dart';
+
+class WindowsWebViewXWidgetState extends State<WebViewXWidget> {
+  late WebviewController _controller;
+  late WindowsWebViewXController _windowsWebViewXController;
+
+  @override
+  void initState() {
+    super.initState();
+
+    initPlatformState();
+  }
+
+  Future<void> initPlatformState() async {
+    _controller = WebviewController();
+
+    await _controller.initialize();
+
+    _windowsWebViewXController = WindowsWebViewXController(
+      _controller,
+      initialContent: widget.initialContent,
+      initialSourceType: widget.initialSourceType,
+      ignoreAllGestures: widget.ignoreAllGestures,
+    );
+
+    if (widget.onWebViewCreated != null) {
+      widget.onWebViewCreated!(_windowsWebViewXController);
+    }
+
+    await _controller.loadUrl('https://flutter.dev');
```

Comment:
Why are we hardcoding https://flutter.dev inside the package?

Reply:
Controller not finished yet, this PR work in progress. I need help to solve the resize issue, that's why I made the PR.

Reply:
I understand, I replied above about the resize issue. You should open a separate issue, and not include it in a pull request :)